ruby_on_rails/rspec.md

```diff
@@ -53,6 +54,10 @@ config.profile_examples = 5
 config.order = :random
 
 Kernel.srand config.seed
+
+config.define_derived_metadata do |meta|
```

Comment:
is there any known disadvantage in globally enabling failure aggregation? i could not find, that it makes it slower so i'm ok with it 👍

Reply:
not that I could find out. of course is "slower", meaning that it does not stop at the first error but it rather continue, therefore is "slower". but only in case of error, so it does not matter IMHO